webapp/loggedin/surveyViews/surveyForm/nodeDefs/components/nodeDefErrorBadge.js

```diff
@@ -63,8 +63,9 @@ const mapStateToProps = (state, props) => {
       if (node) {
         validation = RecordValidation.getNodeValidation(node)(recordValidation)
       } else if (NodeDef.isEntity(nodeDef)) {
-        //only entities can have children with min/max count validation
-        validation = RecordValidation.getValidationChildrenCount(parentNode, nodeDef)(recordValidation)
+        // only entities can have children with min/max count validation
+        validation = Validation.getFieldValidation(parentNode.uuid)(recordValidation)
```

Comment:
how can you distinguish validation errors of different node definitions inside the same parent entity now?

Reply:
and also, i would use our objects model to get the uuid

Reply:
Done both